invoices/update.go

```diff
@@ -196,13 +204,6 @@ func updateMpp(ctx *invoiceUpdateCtx,
 	// Check whether total amt matches other htlcs in the set.
 	var newSetTotal lnwire.MilliSatoshi
 	for _, htlc := range htlcSet {
-		// Only consider accepted mpp htlcs. It is possible that there
```

Comment:
How is this related to the greater fix? 

Do we have a regression test we can commit here, perhaps derived of what you used to originally reproduce the issue?

Reply:
Maybe if a hodl flag was added